lib/ui/views/messenger/layouts/create_talk_sheet.dart

```diff
@@ -37,39 +67,94 @@ class CreateTalkSheet extends ConsumerWidget {
                     bottom: bottom + 80,
                   ),
                   child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
                     children: <Widget>[
                       Column(
                         crossAxisAlignment: CrossAxisAlignment.stretch,
                         children: [
                           TextButton(
-                              onPressed: () {
-                                Sheets.showAppHeightNineSheet(
-                                  context: context,
-                                  ref: ref,
-                                  widget: const CreateGroupSheet(),
-                                );
-                              },
-                              child: Row(
-                                children: [
-                                  Icon(
-                                    Icons.group_add_outlined,
-                                    color:
-                                        theme.textStyleSize14W600Primary.color,
-                                  ),
-                                  const SizedBox(
-                                    width: 8,
-                                  ),
-                                  Text(
-                                    localizations.newGroup,
-                                    style: theme.textStyleSize14W600Primary,
-                                  ),
-                                ],
-                              )),
+                            onPressed: () {
+                              Sheets.showAppHeightNineSheet(
+                                context: context,
+                                ref: ref,
+                                widget: const CreateGroupSheet(),
+                              );
+                            },
+                            child: Row(
+                              children: [
+                                Icon(
+                                  Icons.group_add_outlined,
+                                  color: theme.textStyleSize14W700Primary.color,
+                                ),
+                                const SizedBox(
+                                  width: 8,
+                                ),
+                                Text(
+                                  localizations.newGroup,
+                                  style: theme.textStyleSize14W700Primary,
+                                ),
+                              ],
+                            ),
+                          ),
+                          Divider(
+                            color: theme.textStyleSize14W700Primary.color,
```

Comment:
When you want to use only color, it's better to use directly something like `color: theme.text` from `BaseTheme`interface because in your case, you used a specific style with size (not usefull) and weight (not usefull)